src/bpffeature.cpp

```diff
@@ -401,6 +403,78 @@ bool BPFfeature::has_kprobe_multi()
   return *has_kprobe_multi_;
 }
 
+extern "C" __attribute__((noinline)) __attribute__((unused)) void uprobe_multi_test_func(
+    void)
+{
+}
```

Comment:
I'm afraid this won't work. Packagers typically strip symbols. We've had a lot of issues in the past with BEGIN/END probes being implemented with uprobes inside bpftrace binary. Which is why that moved to raw tracepoint prog type.

Is there another way to feature probe? Perhaps by passing an invalid symbol name and checking for EINVAL (rather than ENOSYS or something).

Reply:
right, actually we can do that, it's better.. I actually did that in libbpf, will change